pkg/services/tree/sync.go

```diff
@@ -200,3 +200,94 @@ func (s *Service) synchronizeSingle(ctx context.Context, d pilorama.CIDDescripto
 		height = newHeight
 	}
 }
+
+// SynchronizeAll forces tree service to synchronize all the trees according to
+// netmap information. Must not be called before Service.Start.
+// Note: blocking operation if the service is already synchronizing.
+func (s *Service) SynchronizeAll() {
+	s.syncChan <- struct{}{}
```

Comment:
What about returning error if this cannot be done immediately? In case the synchronization takes too long we will only make things worse like this.

Reply:
good, added